bleachbit/FileUtilities.py

```diff
@@ -375,17 +375,8 @@ def extended_path(path):
 def free_space(pathname):
     """Return free space in bytes"""
     if 'nt' == os.name:
-        from ctypes import c_ulong, byref, windll
-        if isinstance(pathname, str):
-            pathname = pathname.decode(Common.FSE)
-        freeBytesAvailable = c_ulong()
-        totalNumberOfBytes = c_ulong()
-        totalNumberOfFreeBytes = c_ulong()
-        windll.kernel32.GetDiskFreeSpaceExW(pathname,
-                                            byref(freeBytesAvailable),
-                                            byref(totalNumberOfBytes),
-                                            byref(totalNumberOfFreeBytes))
-        return totalNumberOfFreeBytes.value
+        import psutil
```

Comment:
OK, makes sense. The old method is using `win32file.GetDiskFreeSpaceEx`, but it doesn't accept unicode. Then I switched to `windll.kernel32.GetDiskFreeSpaceExW`, but this sometimes gives different free bytes than `wmic`. So, `windll` is not a viable option either, only `win32file` on non-ascii paths would work. The alternative is to always use `wmic` I guess?